docs/bookstore/page-7/pg7-slack-camel.md

```diff
@@ -0,0 +1,368 @@
+          name: slack-sink
+        properties:
+          channel: ${slack.channel}
+          webhookUrl: ${slack.webhook.url}
+    ```
+
+3. Apply the configuration to your Kubernetes cluster:
+
+```sh
+$ kubectl apply -f slack-sink.yaml
+```
+
+???+ success "Verify"
+      You will see this message if the configuration is created successfully:
+
+      ```sh
+      pipe.camel.apache.org/slack-sink-pipe created
+      ```
+
+      But this process will take a few seconds to complete. You can check the status of the pipe by running the following command:
+
+      ```sh
+      $ kubectl get pipe slack-sink-pipe
+      ```
+
+      ```sh
+      NAME              PHASE     REPLICAS
+      slack-sink-pipe   Ready     1
+      ```
+
+### **Step 4: Modify the Knative Services to disable scale to zero**
+
+![image](images/image6.png)
+
+In this step, we'll configure the notification delivery service to prevent it from scaling down to zero, ensuring timely notifications.
+
+!!! note
+    `ksvc` stands for Knative services.
+
+1. **Check Existing Knative Services:**
+
+```sh
+$ kubectl get ksvcs
+```
+
+You should see a service named `pipe` listed:
+
+```sh
+NAME     URL                                         LATESTCREATED   LATESTREADY    READY   REASON
+pipe     http://pipe.default.svc.cluster.local       pipe-00002      pipe-00002     True
+```
+
+2. **Edit the Knative Service:**
+
+To prevent the notification service from scaling down to zero, set the minimum number of pods to keep running.
+
+```sh
+$ kubectl edit ksvc pipe
+```
+
+Add the following annotation:
+
+```yaml
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/min-scale: "1"
+```
+
+This configuration ensures that Knative will always maintain at least one instance of the service running.
+
+???+ success "Verify"
+
+    ```sh
+    $ kubectl get pods
+    ```
+
+    Periodically check the status of the pipe-deployment pods, and see whether they will disappear! If they stay there, then we are good!
+
+### **Verification**
+
+![image](images/image4.png)
+
+Now, you have completed building the sample app. When you submit a comment, you should always receive a notification in your test Slack workspace, achieving the same result as shown in the demo video.
+
+## **Conclusion**
+
+In this tutorial, you learned how to set up an event-driven service that automates notifications to a Slack channel using Knative Eventing and Apache Camel K. By leveraging these technologies, you can seamlessly connect your applications to third-party services and facilitate real-time information exchange between them.
+
+## **Next Step**
+
+![image](images/image9.png)
+
+Congratulations on successfully completing the bookstore sample app tutorial! If you want to deepen your understanding of Knative, open your bookstore front end, the demo book we used is a great starting point! Check the book "Building Serverless Applications on Knative" by Evan Anderson.
```

Comment:
maybe add a link to https://www.oreilly.com/library/view/building-serverless-applications/9781098142063/